chart/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
If we add our standard file headers, we can remove this file.

Reply:
What does that mean? What are standard file headers?

Is it like this? For each file that is there?

```
// Copyright 2022-2024 The Archivista Contributors
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

// A note: this follows a pattern followed by network service mesh.
// The pattern was copied from the Network Service Mesh Project
// and modified for use here. The original code was published under the
// Apache License V2.
```

Reply:
@matglas Yes, our GHA also checked it.
You can find it in most of our files .go, .yml, .sh, etc.
We skip it only in auto-generated files.